src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnection.cs

```diff
@@ -523,6 +526,19 @@ override public void Open()
         {
             PrepareStatisticsForNewConnection();
 
+            if (StatisticsEnabled ||
```

Comment:
The `PrepareStatisticsForNewConnection` method initializes the _statistics as well. Why is `_statistics` initialization needed again?